client/src/components/File.js

```diff
@@ -2,225 +2,154 @@
 
 import React from 'react';
 import TransitionGroup from "react-addons-css-transition-group";
-import { getHumanReadableBytes } from '../utils/utils.js';
-import apiurl from 'utils/apiurl';
-import 'styles/File.scss';
-import Highlight from 'components/plugins/highlight';
+import Clipboard from 'clipboard';
 import highlight from 'highlight.js'
+import Highlight from 'components/plugins/highlight';
+import { getHumanReadableBytes } from '../utils/utils.js';
 import ChannelActions from 'actions/ChannelActions';
-import Clipboard from 'clipboard';
-
 import MessageStore from 'stores/MessageStore';
-
+import 'styles/File.scss';
 
 import Logger from 'logplease';
 const logger = Logger.create('Clipboard', { color: Logger.Colors.Magenta });
 
-var getFileUrl = apiurl.getApiUrl() + "/api/cat/";
+function readUTF8String(bytes) {
```

Comment:
As per my original comment on this, could you elaborate on *why* we need this conversion? Iirc IPFS return everything "in the correct encoding". Perhaps there's a point somewhere down the line (ie. outside the UI) where the encoding happens/is defined and we can replace all this with something more simple (and this might be related to how getFile() is implemented in Orbit.js.